src/main/java/com/exadel/discount/entity/Country.java

```diff
@@ -0,0 +1,26 @@
+package com.exadel.discount.entity;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.GenericGenerator;
+
+import javax.persistence.*;
+import java.util.List;
+import java.util.UUID;
+
+@Data
+@NoArgsConstructor
+@Entity
+@Table(name = "countries")
+public class Country {
+    @Id
+    @GeneratedValue(generator = "UUID")
```

Comment:
our style is every column naming 
@Column(name = "id")

Reply:
Done